lib/libutee/include/tee_api.h

```diff
@@ -114,7 +114,7 @@ bool TEE_MaskCancellation(void);
 TEE_Result TEE_CheckMemoryAccessRights(uint32_t accessFlags, void *buffer,
 				       uint32_t size);
 
-void TEE_SetInstanceData(void *instanceData);
+void TEE_SetInstanceData(const void *instanceData);
```

Comment:
the "[ctx]" identifier in the specs is expected to get the`const` "attribute" (attribute may not be the right wording). This applies to APIs:
- `TA_OpenSessionEntryPoint()`
- `TA_CloseSessionEntryPoint()`
- `TA_InvokeCommandEntryPoint()`
- `TEE_SetInstanceData()`
- `TEE_GetInstanceData()`